server/src/routes/staff.router.ts

```diff
@@ -0,0 +1,17 @@
+import express from "express";
+
+import {
+  createStaff,
+  getAllStaff,
+  getStaffById,
+  getStaffByFirebaseId,
+} from "../controllers/staffmodel.controller";
+
+const staffRoute = express.Router();
+
+staffRoute.post("/", createStaff);
+staffRoute.get("/allstaff", getAllStaff);
+staffRoute.get("/getstaff", getStaffById);
+staffRoute.get("/", getStaffByFirebaseId);
```

Comment:
Let's make this line `staffRoute.get("/firebase/:firebaseUID", getStaffByFirebaseId);`